aten/src/ATen/native/README.md

```diff
@@ -279,13 +279,19 @@ to reuse the same function name in both cases.
 
 Available backend options can be found at
 https://github.com/pytorch/pytorch/blob/master/tools/codegen/gen.py#L970.
-In addition to backends above, we also support keyword `Math` which is an alias
-that maps to all backend and autograd backend keys. In other words, function registered to `Math` key
-should be a plain mathematical composition of other `at::` functions and works for any backend.
+In addition to backends above, we also support keywords:
+  - `DefaultBackend`: an alias that maps to all backends. Functions registered to
+    `DefaultBackend` should work for any backend inference.
+  - `Math`: an alias that maps to all backend and autograd backend keys. Function
+    registered to `Math` key should be a plain mathematical composition of other
+    `at::` functions and supports training and inference for any backend.
```

Comment:
you are referring to the function here, which should be plural, which means supports here should just be "support"